content/flux/v0.x/query-data/bigtable.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Query Google Cloud Bigtable
+list_title: Google Cloud Bigtable
+description: >
+  Use [`bigtable.from`](/flux/v0.x/stdlib/experimental/bigtable/from) to
+  query [Google Cloud Bigtable](https://cloud.google.com/bigtable/) with Flux.
+menu:
+  flux_0_x:
+    name: Bigtable
+    parent: Query data sources
+weight: 104
+related: 
+  - /flux/v0.x/stdlib/experimental/bigtable/from/
+  - https://cloud.google.com/bigtable/docs/access-control, Cloud Bigtable Access Control
+  - https://cloud.google.com/iam/docs/how-to, Google Cloud IAM How-to guides
+  - https://cloud.google.com/docs/authentication/production, Setting Up Authentication for Production Applications on Google Cloud
+list_code_example: |
+  ```js
+  import "experimental/bigtable"
+  
+  bigtable.from(url: "http://example.com/metrics")
+  ```
+---
+
+To query [Google Cloud Bigtable](https://cloud.google.com/bigtable/) with Flux:
+
+1. Import the [`experimental/bigtable` package](/flux/v0.x/stdlib/experimental/bigtable/).
```

Comment:
```suggestion
1.  In [InfluxDB Data Explorer]((/{{< latest "influxdb" >}}/query-data/execute-queries/data-explorer/) or the [Flux REPL], import the [`experimental/bigtable` package](/flux/v0.x/stdlib/experimental/bigtable/).
```

Reply:
@sanderson would the shortcode above work for both cloud and v2.0 context?

Reply:
There are a lot of different ways they could execute this query (API, data explorer, REPL, client library, etc.). That's why I don't specifically call it out here.